src/content/docs/quick-tips/send-osc.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Sending OSC
+description: Sending OSC (Open Sound Control) from Ontime
+---
+
+You can leverage the [integrations feature](/api/integrations) to send Ontime data to other applications via OSC (Open Sound Control).
+
+
+## OSC Payload examples
+When sending an OSC message, the argument type is not explicitly given by the user but inferred by Ontime.
+
+
+```
+| OSC Payload                   | Parsed type                   |
+| ----------------------------- | ----------------------------- |
+| 1111                          | Int                           |
+| 0.1111                        | Float                         |
+| test                          | String                        |
+| "1111"                        | String                        |
+| true                          | True                          |
+| false                         | False                         |
+```
+
+You can also send multiple, space separated arguments which will be parsed individually.
+
+```bash title="Multiple arguments"
+"with space" 1111 0.1111 TRUE FALSE
+```
+
+Which would be sent as a single message with the payload
+
+```
+| Payload value                 | Parsed type                   |
+| ----------------------------- | ----------------------------- |
+| 'with space'                  | String                        |
+| 1111                          | Int                           |
+| 0.1111                        | Float                         |
+| true                          | True                          |
+| false                         | False                         |
+```
```

Comment:
```suggestion
```

Keep in mind this also affects variables so if you add in {{eventNow.title}} it would be wise to wrap it in quotes in case there is a space in the title like so "{{eventNow.title}}"

also {{eventNow.cue}} is often a number string and will be converted to float or int automatically, to force it to a string you can also wrap it in quotes  "{{eventNow.cue}}"

if you have something that can not handle that numbers could be both integers or floats you can force it to be float by adding a 0 behind a decimal point 

0 will be sendt as int
0.0 will be send as float

```

something like that, you can probably formulate it better

Reply:
this first case seems potentially tricky

> Keep in mind this also affects variables so if you add in {{eventNow.title}} it would be wise to wrap it in quotes in case there is a space in the title like so "{{eventNow.title}}"

I wonder if we should be the ones injecting the ""? I think it is safe to assume that in no case the user wants a dynamic payload for every word.

Reply:
then we would need to know the type otherwise we would also force all int and floats to be strings

Reply:
Checking the type inside our app is pretty trivial. All our fields are typed. Am I missing something?

Reply:
we don't get the types out of the template parser, everything is strings
but maybe we could rework that to give it?

Reply:
we probably need to consider this carefully

for example

`{{event.start}}` should probably return a number, this would be a trivial change to the parser
but what happens when we want to do next `event starts in {{event.start}}`?

We need to be careful not to end up with two arguments like `event starts in` and `{{event.start}}`
I think this may not be too complex, but we should look into it carefully as we go along

Reply:
should I commit some changes?

Reply:
@alex-Arc , you are welcome to propose changes